notebooks/official/feature_store/vertex_ai_feature_store_feature_view_service_agents.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<ol><li>Learn more about <a href="https://cloud.google.com/vertex-ai/docs/featurestore/overview" target="_blank">Vertex AI Feature Store</a> leads users to the Feature store legacy. Is it what we want?</li><li>Typo tuturial &gt; tutorial</li><li>"Users need to manually call ...": Suggest to remove "manually". This may implicit to run the command manually outside of this notebook.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Fixed.</p>